src/grd2cpt.c

```diff
@@ -421,10 +421,14 @@ static int parse (struct GMT_CTRL *GMT, struct GRD2CPT_CTRL *Ctrl, struct GMT_OP
 		}
 	}
 
-	if (Ctrl->H.active && GMT->current.setting.run_mode == GMT_CLASSIC) {
-		n_errors++;
-		GMT_Report (GMT->parent, GMT_MSG_ERROR, "Unrecognized option -H\n");
+	if (GMT->current.setting.run_mode == GMT_CLASSIC) {
+		if (Ctrl->H.active) {
+			n_errors++;
+			GMT_Report (GMT->parent, GMT_MSG_ERROR, "Unrecognized option -H\n");
```

Comment:
It would be better to say that "-H option is not valid in classic mode". Or even better, just ignore it silently.

Reply:
OK, and I changed it to a warning.  Not a fan of silence for wrong use.

Reply:
OK, but remember that sometimes the commands were not written by humans.